packages/flutter/test/material/popup_menu_test.dart

```diff
@@ -690,6 +690,81 @@ void main() {
     await tester.tap(find.text('Menu Button'));
   });
 
+  testWidgets('PopupMenuItem child height is a minium, child is vertically centered', (WidgetTester tester) async {
+    // This is a regression test for https://github.com/flutter/flutter/issues/39508.
```

Comment:
This isn't a regression test since it was always this way (as far as I can tell from the linked issue). Should we still link an issue in a comment like this if it it's a bug fix vs a regression?

Reply:
Good point; the new tests aren't really regression tests, since the "fix" was to change the meaning of the popup menu item's height parameter.